src/pages/MyPage.js

```diff
@@ -1,5 +1,5 @@
-import React from "react";
-import HeaderContainer from "../components/HeaderContainer";
+import React from 'react';
+import HeaderContainer from '../components/HeaderContainer';
 
 export default function MyPage() {
   return <HeaderContainer></HeaderContainer>;
```

Comment:
return에 소괄호 넣고 줄바꿈 해주세요~
```suggestion
  return (
      <HeaderContainer name={"test"} />;
  );
```
이런 식으로 하면 될 것 같아요